src/client/clientlauncher.cpp

```diff
@@ -211,7 +211,8 @@ bool ClientLauncher::run(GameStartData &start_data, const Settings &cmd_args)
 		m_rendering_engine->get_raw_device()->
 			setWindowCaption((utf8_to_wide(PROJECT_NAME_C) +
 			L" " + utf8_to_wide(g_version_hash) +
-			L" [" + wstrgettext("Main Menu") + L"]").c_str());
+			L" [" + wstrgettext("Main Menu") + L"]" +
+			L" [" + m_rendering_engine->getVideoDriver()->getName() + L"]"	).c_str());
```

Comment:
Nitpick: With opengl3 driver, this doesn't include "OpenGL" for me, i.e. it's just "4.6 (Compatibility Profile) Mesa 23.1.2".

Reply:
That’s just what Irrlicht returned.